include/swift/SwiftRemoteMirror/SwiftRemoteMirrorTypes.h

```diff
@@ -265,8 +268,15 @@ typedef struct swift_actor_info {
   /// swift_reflection call on the given context.
   const char *Error;
 
-  uint64_t Flags;
+  uint8_t State;
+  uint8_t IsDistributedRemote;
+  uint8_t IsPriorityEscalated;
```

Comment:
Should these 2 really be `uint8_t` rather than bit flags (or `_Bool`)?

Reply:
Space isn't a concern, but using a boolean would be smart. C++ doesn't like `_Bool` but `bool` plus `stdbool.h` does the job.